app/ide-desktop/lib/client/src/authentication.ts

```diff
@@ -156,38 +156,49 @@ export function onOpenUrl(url: URL, window: () => electron.BrowserWindow) {
-        /** System agnostic credentials directory home path. */
-        const credentialsHomePath = path.join(os.homedir(), credentialsDirectoryName)
+            /** Home directory for the credentials file.  */
+            const credentialsDirectoryName = `.${common.PRODUCT_NAME.toLowerCase()}`
+            /** File name of the credentials file. */
+            const credentialsFileName = 'credentials'
+            /** System agnostic credentials directory home path. */
+            const credentialsHomePath = path.join(os.homedir(), credentialsDirectoryName)
 
-        if (accessToken == null) {
-            try {
-                fs.unlinkSync(path.join(credentialsHomePath, credentialsFileName))
-            } catch {
-                // Ignored, most likely the path does not exist.
-            }
-        } else {
-            fs.mkdir(credentialsHomePath, { recursive: true }, error => {
-                if (error) {
-                    logger.error(`Couldn't create ${credentialsDirectoryName} directory.`)
-                } else {
-                    fs.writeFile(
-                        path.join(credentialsHomePath, credentialsFileName),
-                        accessToken,
-                        innerError => {
-                            if (innerError) {
-                                logger.error(`Could not write to ${credentialsFileName} file.`)
-                            }
-                        }
-                    )
+            if (accessTokenPayload == null) {
+                try {
+                    fs.unlinkSync(path.join(credentialsHomePath, credentialsFileName))
+                } catch {
+                    // Ignored, most likely the path does not exist.
                 }
-            })
+            } else {
+                fs.mkdir(credentialsHomePath, { recursive: true }, error => {
+                    if (error) {
+                        logger.error(`Couldn't create ${credentialsDirectoryName} directory.`)
+                    } else {
+                        fs.writeFile(
+                            path.join(credentialsHomePath, credentialsFileName),
+                            JSON.stringify({
+                                /* eslint-disable @typescript-eslint/naming-convention */
+                                client_id: accessTokenPayload.clientId,
+                                access_token: accessTokenPayload.accessToken,
+                                refresh_token: accessTokenPayload.refreshToken,
+                                refresh_uri: accessTokenPayload.refreshUri,
+                                expire_at: accessTokenPayload.expireAt,
+                                /* eslint-enable @typescript-eslint/naming-convention */
+                            }),
+                            innerError => {
+                                if (innerError) {
+                                    logger.error(`Could not write to ${credentialsFileName} file.`)
```

Comment:
this also existed before but...  we normally quote all dynamic strings in `''`:
```ts
`Could not write to the file '${credentialsFileName}'.`
```

also i think it might be more useful to print the file path instead of the file name